modules/webrtc/webrtc_multiplayer_peer.h

```diff
@@ -69,7 +69,7 @@ class WebRTCMultiplayerPeer : public MultiplayerPeer {
 	int next_packet_peer = 0;
 	bool server_compat = false;
 
-	Map<int, Ref<ConnectedPeer>> peer_map;
+	HashMap<int, Ref<ConnectedPeer>> peer_map;
```

Comment:
This is used to also balance incoming packets fairly among peers (in `_fin_next_peer`)
So I guess it shold be an `RBSet`/`RBMap`?

Reply:
Not really I guess, it's more about keeping a consistent order.
That said though, given the number of peers doesn't change often (so the order won't change often), it's actually probably okay to stay as it is now.